OOP/Assignment 1/OOPQuestion1/Day5TestJUnit.java

```diff
@@ -0,0 +1,40 @@
+import java.util.HashMap;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+public class Day5TestJUnit
+{
+
+	@Test
+	public void test()
+	{
+		ServiceStation serviceStation = new ServiceStation();
+		serviceStation.readMechanics();
+		serviceStation.readCars();
+		serviceStation.doAllotment();
+		HashMap<String, Integer> carProcessingList = serviceStation.getCarProcessingStatus();
```

Comment:
As per coding convention, always use reference variable of type Interface like List, Map, Set and create an object of Sub-class like arrayList, linkedlist, hashset, linkedhashset, hashmap and so on.
Map<String, Integer> carProcessingMap = serviceStation.getCarProcessingStatus();